### PR TITLE
feat: peers page

### PR DESCRIPTION
**Pull request by fsdiogo**
Tasks:
- [x] World map
- [x] Peers table ([react-virtualized](https://github.com/bvaughn/react-virtualized))
- [x] Dynamic locations in the map 
- [x] Flags in the table
- [x] Merge https://github.com/ipfs-shipyard/ipfs-webui/pull/702 first

<br>

Closes https://github.com/ipfs-shipyard/ipfs-webui/issues/700.

### CODE REVIEW

**Comment by fsdiogo** [comment]
@hacdias Slow when scrolling or fetching the flags? I'm using `react-virtualized` to only render the visible rows, but you can scroll to see more. 

I can add the margin, but it was not supposed to appear `US` there, only flags 😅

**Comment by hacdias** [comment]
@fsdiogo it seems that Windows Flag emojis are just the country code sadly. Yesterday it seemed more slower. When I resize the page and scroll it seems a bit slow

**Comment by fsdiogo** [comment]
@hacdias ohh, so the flags aren't rendering on Windows? 😓

Can you test the following please (I don't have a windows machine):

In the `src/peers/PeersTable/PeersTable.js`, line `14`, can you change that to:

```js
{ rowData.flagCode ? <CountryFlag code={rowData.flagCode} svg /> : '🏳️‍🌈' }
```

That way we are force rendering the flags to SVG instead of emoji. Maybe it will work.

**Comment by hacdias** [comment]
@fsdiogo 

Yes, that way they render:

image

Still, we need some margin there

**Comment by hacdias** [comment]
@fsdiogo merged the #702

**Comment by fsdiogo** [comment]
@hacdias awesome, thanks!

It doesn't look as good as with the emojis though:

screen shot 2018-07-19 at 11 04 05

We have two options here:
1. Always render the flags as SVG
2. Use something like [platform.js](https://github.com/bestiejs/platform.js/) to detect if it's Windows, and render as SVG accordingly

What do you think?

**Comment by hacdias** [comment]
Yeah, the emojis are better even though they are platform dependent. You can render with SVGs on Windows or let it use the Windows "emojis". If we render SVGs conditionally, I don't think there's a need to add a dependency when we can just:

```js
svg={window.navigator.appVersion.indexOf('Win') !== -1}
```

File: src/bundles/peers.js [hunk 1]
```diff
@@ -9,6 +9,43 @@ const bundle = createAsyncResourceBundle({
   checkIfOnline: false
 })
 
+bundle.selectTableData = state => {
+  const peers = state.peers.data
+
+  return peers && peers.map((peer, idx) => ({
+    'id': peer.peer.toB58String(),
+    'address': peer.addr.toString(),
+    'location': 'New York, United States'
```
Comment (hacdias) [inline src/bundles/peers.js]: Why's New York here?

Comment (fsdiogo) [reply src/bundles/peers.js]: Oops, that selector is not being used, I made a new below. I'll commit a fix.

File: src/peers/PeersTable/PeersTable.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+.PeersTableContainer {
+  height: 220px;
```
Comment (olizilla) [inline src/peers/PeersTable/PeersTable.css]: Minor, but I'd just use an inline style attributes for this as the value is duplicated in the definition of the table. I'd inline `.MapContainer` as well, just to keep all the definitions together.

Comment (fsdiogo) [reply src/peers/PeersTable/PeersTable.css]: Agree, and that way we can delete those two CSS files 👍

**Comment by fsdiogo** [comment]
Can we merge this? 🚀

**Comment by hacdias** [comment]
@fsdiogo here we go 😄

**Pull request merged**